workers/src/main/scala/workers/WorkerServer.scala

```diff
@@ -109,4 +110,12 @@ trait WorkerServer[F[_]: Async, T: Pickler](using Monoid[F[Unit]]) {
             .handleErrorWith(e => Logger[F].error(e)("Error processing message in worker"))
         )
     ).handleErrorWith(e => Logger[F].error(e)("Error initializing worker"))
-}
+
+  protected def runInternal(dispatcher: Dispatcher[F])(using Logger[F]): F[Unit] =
+    for {
+      self         <- F.delay(dom.DedicatedWorkerGlobalScope.self)
+      handlerFn    <- handler
+      cancelTokens <- Ref[F].of(Map.empty[WorkerProcessId, F[Unit]])
+      _            <- mount(self, handlerFn, cancelTokens)(dispatcher)
+      _            <- postAsTransferable[F, FromServer](self, FromServer.Ready)
```

Comment:
Will clients wait for this message?

Reply:
Yup, it's done here https://github.com/gemini-hlsw/explore/blob/move-to-ui/common/src/main/scala/workers/WorkerClient.scala#L27